tests/langchain/test_langchain_autolog.py

```diff
@@ -786,3 +786,37 @@ def test_langchain_autolog_callback_injection_in_batch(invoke_arg, generate_conf
         assert sorted(callbacks.handlers[0].logs) == expected_logs
     else:
         assert sorted(callbacks[0].logs) == expected_logs
+
+
+@pytest.mark.parametrize("invoke_arg", ["args", "kwargs"])
+@pytest.mark.parametrize(
+    "generate_config",
+    [
+        lambda: RunnableConfig(callbacks=[CustomCallbackHandler()]),
+        lambda: RunnableConfig(callbacks=BaseCallbackManager([CustomCallbackHandler()])),
+    ],
+)
+def test_langchain_autolog_callback_injection_in_stream(invoke_arg, generate_config):
+    mlflow.langchain.autolog(log_models=True, extra_tags={"test_tag": "test"})
+    config = generate_config()
+    with mlflow.start_run() as run, _mock_request(return_value=_mock_chat_completion_response()):
+        model = create_openai_llmchain()
+        # convert iterator to list to materialize the stream compute
+        if invoke_arg == "args":
+            list(model.stream("MLflow", config))
```

Comment:
```suggestion
            next(model.stream("MLflow", config))
```

Is `next` enough?

Reply:
`list` does the same but `next` sounds more like `advance the iterator`.

Reply:
Yep agree!